config/samples/gatling-operator_v1alpha1_gatling01.yaml

```diff
@@ -3,10 +3,12 @@ kind: Gatling
 metadata:
   name: gatling-sample01
 spec:
-  generateReport: false                                   # The flag of generating gatling report
+  generateReport: true                                   # The flag of generating gatling report
+  #generateReport: false                                   # The flag of generating gatling report
   generateLocalReport: false                              # The flag of generating gatling report for each pod
   notifyReport: false                                     # The flag of notifying gatling report
-  cleanupAfterJobDone: true                               #  The flag of cleaning up gatling jobs resources after the job done
+  cleanupAfterJobDone: false                               #  The flag of cleaning up gatling jobs resources after the job done
+  #cleanupAfterJobDone: true                               #  The flag of cleaning up gatling jobs resources after the job done
```

Comment:
In this sample, I think `cleanupAfterJobDone` is intended to be true, is it not?

Reply:
you're light!  it's just for testing. https://github.com/st-tech/gatling-operator/pull/59/commits/c49c5091eca5c3ca5a0f5e77d17667e2c98acf21